pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -1061,12 +1061,12 @@ func isPodResourcesResizedDown(pod *v1.Pod) bool {
 
 // AssignedPodUpdated is called when a bound pod is updated. Change of labels
 // may make pending pods with matching affinity terms schedulable.
-func (p *PriorityQueue) AssignedPodUpdated(logger klog.Logger, oldPod, newPod *v1.Pod) {
+func (p *PriorityQueue) AssignedPodUpdated(logger klog.Logger, oldPod, newPod *v1.Pod, event framework.ClusterEvent) {
 	p.lock.Lock()
 	if isPodResourcesResizedDown(newPod) {
```

Comment:
The event already has the information about whether there was a change of requests

Reply:
Updated.